plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/utils/ServerMessageUtil.java

```diff
@@ -57,35 +57,6 @@ public static ServerMessageAction error(final Exception e) {
       return error(getMessage(e), getDetails(e));
    }
 
-   public static ServerMessageAction message(final Severity severity, final String message,
-      final int timeout) {
```

Comment:
Any specific reason why we remove this entirely instead of just removing the timeout parameter?

Reply:
I think because they are already above, if I'm not mistaken?

Reply:
Ah makes sense. I didn't look at the full scope of the file 😄